db/entity/src/conversions/channels.rs

```diff
@@ -109,21 +116,23 @@ impl TryFrom<channel::Model> for CorruptedChannelEntry {
     }
 }
 
+impl From<ChannelId> for channel::ActiveModel {
+    fn from(value: ChannelId) -> Self {
+        channel::ActiveModel {
+            channel_id: Set(value.to_hex()),
+            source: Set(Address::default().to_hex()), // Default to empty address
+            destination: Set(Address::default().to_hex()), // Default to empty address
+            balance: Set(HoprBalance::default().amount().to_be_bytes().into()), //
+            status: Set(i8::from(ChannelStatus::Open)), // Default to Open status
+            ..Default::default()
```

Comment:
I would rather move this code into the `impl From<CorruptedChannelEntry> for channel::ActiveModel`, we don't need this elsewhere.